### PR TITLE
Add a community strategy sub-page and provide a brief update on works so far

### PR DESCRIPTION
**Pull request by sgibson91**
Adds a sub-page to our docs to gather information on the CSL role. Provides a brief update on thinkings/activities so far.

This is super bare bones to begin with. Happy to tweak format as needed.

### CODE REVIEW

File: docs/team/community-strategy.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Community Strategy
+
+This page contains information regarding activities undertaken and community strategies developed by the Community Strategic Lead.
+[Sarah Gibson](https://github.com/sgibson91) is currently serving in this role as of August 2022, for a period of two years.
+
+This role and related activities are funded by a CZI EOSS DEI grant, administered by NumFOCUS.
```
Comment (choldgraf) [inline docs/team/community-strategy.md:6]:
```suggestion
This role and related activities are funded by [a CZI EOSS DEI grant](https://drive.google.com/file/d/124LWcFe8Hq2n_3l4jAgDgJ1lpXRzV1Kc/view?usp=sharing), administered by [NumFOCUS](https://numfocus.org).
```

File: docs/team/community-strategy.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+```{admonition} Work in Progress
+The work represented on this page is under active development and we will update information as soon as we are able to as we iterate.
+```
+
+Some information is being shared in [Issue #536](https://github.com/jupyterhub/team-compass/issues/536), which will be migrated here over time.
```
Comment (choldgraf) [inline docs/team/community-strategy.md:12]: The PyData theme will soon release support for lightly auto-formatting github issue/PR links, so we could just put in the raw link if we like: https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/theme-elements.html#link-shortening-for-git-repository-services

File: docs/team/community-strategy.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+Additionally, you can find information on other related activities via the [![csl-label](https://img.shields.io/github/labels/jupyterhub/team-compass/community-strategic-lead)](https://github.com/jupyterhub/team-compass/issues?q=label%3Acommunity-strategic-lead) label.
+
+## Community Strategy Updates
+
+### 2022-10-07: 2 month strategy so far
+
+One strategy is to develop onboarding pathways through Outreachy,then into Contributor in Residence, and finally as a fully-fledged community member.
```
Comment (choldgraf) [inline docs/team/community-strategy.md:19]: Let's flesh this out in a future update - we can brainstorm via these updates, and then encode things in the team compass via more focused PRs

File: docs/team/community-strategy.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+This role and related activities are funded by a CZI EOSS DEI grant, administered by NumFOCUS.
+
+```{admonition} Work in Progress
+The work represented on this page is under active development and we will update information as soon as we are able to as we iterate.
+```
+
+Some information is being shared in [Issue #536](https://github.com/jupyterhub/team-compass/issues/536), which will be migrated here over time.
+Additionally, you can find information on other related activities via the [![csl-label](https://img.shields.io/github/labels/jupyterhub/team-compass/community-strategic-lead)](https://github.com/jupyterhub/team-compass/issues?q=label%3Acommunity-strategic-lead) label.
+
+## Community Strategy Updates
```
Comment (choldgraf) [inline docs/team/community-strategy.md:15]: Eventually we probably want a dedicated page for these (e.g. a "Changelog" for the CSL position) but I think it's fine here for now

**Review by choldgraf — APPROVED** [review]
This looks good to me - over time we might want to split this out into multiple pages but IMO we should merge and iterate from there. I made a few suggestions here and there, feel free to merge whenever you like though.

File: docs/team/community-strategy.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+### 2022-10-07: 2 month strategy so far
```
[the remark this answers is not in the document]
Comment (sgibson91) [reply docs/team/community-strategy.md:17]: Nope, this is a "Sarah can't write down dates properly" date

Comment (sgibson91) [inline docs/team/community-strategy.md:17]:
```suggestion
### 2022-09-07: 2 month strategy so far
```

[the remark this answers is not in the document]
Comment (sgibson91) [reply docs/team/community-strategy.md]: Fixed in [`f3992b0` (#560)](https://github.com/jupyterhub/team-compass/pull/560/commits/f3992b0f7772c54d295b580c29f3b3ce5c7826b6)

File: docs/team/community-strategy.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+This page contains information regarding activities undertaken and community strategies developed by the Community Strategic Lead.
+[Sarah Gibson](https://github.com/sgibson91) is currently serving in this role as of August 2022, for a period of two years.
+
+This role and related activities are funded by [a CZI EOSS DEI grant](https://drive.google.com/file/d/124LWcFe8Hq2n_3l4jAgDgJ1lpXRzV1Kc/view?usp=sharing), administered by [NumFOCUS](https://numfocus.org).
+
+```{admonition} Work in Progress
+The work represented on this page is under active development and we will update information as soon as we are able to as we iterate.
+```
+
+Some information is being shared in [Issue #536](https://github.com/jupyterhub/team-compass/issues/536), which will be migrated here over time.
```
Comment (sgibson91) [inline docs/team/community-strategy.md:12]:
```suggestion
<!-- Using a raw link here as a PyData Sphinx theme release will format them for us: https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/theme-elements.html#link-shortening-for-git-repository-services -->
Some information is being shared in https://github.com/jupyterhub/team-compass/issues/536, which will be migrated here over time.
```

File: docs/team/community-strategy.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+<!-- Using a raw link here as a PyData Sphinx theme release will format them for us: https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/theme-elements.html#link-shortening-for-git-repository-services -->
+Some information is being shared in https://github.com/jupyterhub/team-compass/issues/536, which will be migrated here over time.
+Additionally, you can find information on other related activities via the [![csl-label](https://img.shields.io/github/labels/jupyterhub/team-compass/community-strategic-lead)](https://github.com/jupyterhub/team-compass/issues?q=label%3Acommunity-strategic-lead) label.
+
+## Community Strategy Updates
+
+### 2022-09-07: 2 month strategy so far
+
+One strategy is to develop onboarding pathways through Outreachy,then into Contributor in Residence, and finally as a fully-fledged community member.
```
Comment (sgibson91) [inline docs/team/community-strategy.md:20]:
```suggestion
One strategy is to develop onboarding pathways through Outreachy, then into Contributor in Residence, and finally as a fully-fledged community member.
```

**Pull request merged**